src/shogun/clustering/Hierarchical.h

```diff
@@ -108,6 +108,11 @@ class CHierarchical : public CDistanceMachine
 			return false;
 		}
 
+		/** TODO: Ensures cluster centers are in lhs of underlying distance
```

Comment:
could you open an issue with this?

Reply:
this comment as left before, i just copy-pasted to make the method public